test/Microsoft.IdentityModel.Tests/IdentityUtilities.cs

```diff
@@ -93,6 +94,99 @@ public static JwtSecurityToken CreateJwtSecurityToken(string issuer, string audi
             return new JwtSecurityToken(header, payload, header.Base64UrlEncode(), payload.Base64UrlEncode(), "" );
         }
 
+        public static string CreateSamlSecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
+        {
+            SecurityToken securityToken = tokenHandler.CreateToken(securityTokenDescriptor);
+            StringBuilder sb = new StringBuilder();
+            XmlWriter writer = XmlWriter.Create(sb);
+            tokenHandler.WriteToken(writer, securityToken);
+            writer.Flush();
+            writer.Close();
+            return sb.ToString();
+        }
+
+        public static string CreateSaml2SecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
+        {
+            SecurityToken securityToken = tokenHandler.CreateToken(securityTokenDescriptor);
+            StringBuilder sb = new StringBuilder();
+            XmlWriter writer = XmlWriter.Create(sb);
+            tokenHandler.WriteToken(writer, securityToken);
+            writer.Flush();
+            writer.Close();
+            return sb.ToString();
+        }
+
+        public static string CreateJwtSecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
+        {
+            SecurityToken securityToken = tokenHandler.CreateToken(securityTokenDescriptor);
+            return tokenHandler.WriteToken(securityToken);
+        }
+
+        public static SecurityTokenDescriptor DefaultX509SecurityTokenDescriptor
```

Comment:
A lot of these 'Default' methods are duplicates of what is in Default.cs. 
Remove them and If something is missing Default.cs, add the property there.

Reply:
Fixed